src/ImageSharp/IO/ChunkedMemoryStream.cs

```diff
@@ -555,11 +557,13 @@ private sealed class MemoryChunk : IDisposable
     {
         private bool isDisposed;
 
-        public IMemoryOwner<byte> Buffer { get; set; }
+        public MemoryChunk(IMemoryOwner<byte> buffer) => this.Buffer = buffer;
```

Comment:
At first I did not want to create a constructor for this because there was none..

I wanted to use the required keyword, but I could not use it because it is a C#11 feature.

That's why I went with the constructor

Reply:
That's all cool. I had to do the same with `TextOptions` in the Fonts library.